test/plugins/addClassesToSVGElement.02.svg

```diff
@@ -0,0 +1,13 @@
+<svg xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please add description

```
description

===

<svg...
```

Reply:
I just copied and adapted the other addClassesToSVGElement test file